utils/updateFileApproval.ts

```diff
@@ -0,0 +1,36 @@
+import { Student } from "@prisma/client";
+import { mutate } from "swr";
+
+/**
+ * updateFileApproval updates a file's approval from an admin
+ * @param filePath
+ * @param email
+ * @param approval
+ * @returns deleted records
+ */
+export async function updateFileApproval(
+    filePath: string,
+    email: string,
```

Comment:
I think it'd be easier for us to use ids instead since we are playing to use dynamic routes for these registrant pages via their user ids.
```suggestion
    id: number, 
```

we'll need to change the logic below accordingly.